### PR TITLE
[#11794] Update help pages

### LINKED ISSUE #11794: Help pages need some updates
Creating this issue to update the instructor help page, since the changes are not targeting issues #2874 and #8769 (those are about adding questions to the instructor help page). There would be a separate PR for those issues. More details of the updates are posted in the Slack channel.

### PR DESCRIPTION
**Pull request by fans2619**
Fixes #11794

### CODE REVIEW

**Comment by fans2619** [comment]
image

**Comment by fans2619** [comment]
image

**Comment by damithc** [comment]
> image

@fans2619 Perhaps explain both options? e.g.,

> ... then select one of these options:
> * **View Results (from/to me)**: to view responses you submitted, and responses others submitted about you
>* **View Results (course wide)**: to view all responses collected by that session

**Comment by damithc** [comment]
> image

Looks OK to me, but you'll probably need @samuelfangjw 's view on this as he is familiar with this new feature more than me.

**Comment by fans2619** [comment]
> Perhaps explain both options? e.g.,
> 
> > ... then select one of these options:
> > 
> > * **View Results (from/to me)**: to view responses you submitted, and responses others submitted about you
> > * **View Results (course wide)**: to view all responses collected by that session

Hmm although explaining both looks better, the original intention of this question seems to be only "view all responses collected by the session". This question is linked to by a few other questions and all of them seem to refer to the "view all responses" option as well.

**Comment by damithc** [comment]
> Hmm although explaining both looks better, the original intention of this question seems to be only "view all responses collected by the session". This question is linked to by a few other questions and all of them seem to refer to the "view all responses" option as well.

True. It's because the `from/to me` options was added more recently. Is it mentioned anywhere else in  the help page? If not, probably good enough to explain it in this question itself? Any other suggestions?

**Comment by fans2619** [comment]
image

**Comment by samuelfangjw** [comment]
> image

Seems fine to me! Some changes to consider:
For 2, we can add 'Students who already have deadline extensions are highlighted in yellow'
For 3, we might want to consider expanding on this and explain the following:
- The two ways we can extend (extend by and extend to) 
- How checking `Notify affected students and/or instructors by email about this extension` will send an email to affected students/instructors regarding the extension.

**Comment by fans2619** [comment]
image

Personally I find it a little wordy and it may not be necessary to explain every detail when the UI can be self-explanatory.

**Comment by samuelfangjw** [comment]
> image
> 
> Personally I find it a little wordy and it may not be necessary to explain every detail when the UI can be self-explanatory.

Disagree. By this logic, there isn't a need to document anything at all. A simple 'Yes you can extend deadlines from the edit session page' will be sufficient. It's all self explanatory after all.

The level of detail requested is not more than many of the other questions in the help page.

**Comment by fans2619** [comment]
> The level of detail requested is not more than many of the other questions in the help page.

Ah I agree with this. I'm actually not only targeting this question, but think the current help page contains more details than needed in general. It's like a User Guide instead of a normal help page, but probably that's the intention.

**Comment by damithc** [comment]
> It's like a User Guide instead of a normal help page, but probably that's the intention.

Yes, it also serves as the user guide ATM. So, it's going to be a bit wordy but of course we can skip trivial or self-evident info.

**Comment by fans2619** [comment]
image

---

image

**Comment by fans2619** [comment]
image
image

**Comment by fans2619** [comment]
Kindly see the commits and commit messages to understand what changes have been made when reviewing.

**Review by jianhandev — APPROVED** [review]
Good work, LGTM.

**Review by wkurniawan07 — APPROVED** [review]
LGTM
